### PR TITLE
add support for ConstantOfShape, Cos, Div, Equal, Expand, Neg, Pow, Transpose, and Where

### PR DESCRIPTION
**Pull request by dstarkenburg**
Hello,

My name is Duncan I am a student at the University of Vermont. I am working as a research assistant under Sam Chevalier in the Department of Electrical Engineering. We are working on large-scale verification problems in Julia and would love to make ONNX.jl a finished package (or at least work towards it).

In this pull request I've added support for the following graph layer types:
- ConstantOfShape
- Cos
- Div
- Equal
- Expand
- Neg
- Pow
- Transpose
- Where

I also ordered each load, save, and operation alphabetically in their respective files (to match the ONNX documentation and keep track of missing layer types).

I have not, however, created a testing suite for these layers but have tested them on ONNX graphs.

Please let me know how to proceed; I would love to continue working on ONNX.jl!

- Duncan

### PR Checklist

- [NO] Tests are added
- [YES] Documentation, if applicable

### CODE REVIEW

File: src/save.jl [hunk 1]
```diff
@@ -148,71 +198,84 @@ function save_node!(g::GraphProto, ::@opconfig_kw(:ONNX, conv), op::Umlaut.Call)
...
-function save_node!(g::GraphProto, ::OpConfig{:ONNX, typeof(global_average_pool)}, op::Umlaut.Call)
-    nd = NodeProto("GlobalAveragePool", op)
+# Equal
+function save_node!(g::GraphProto, ::OpConfig{:ONNX, typeof(equal)}, op::Umlaut.Call)
+    nd = NodeProto("Equal", op)
     push!(g.node, nd)
 end
 
+# Expand
+function load_node!(tape::Tape, ::OpConfig{:ONNX, :Expand}, args::VarVec, attrs::AttrDict)
```
Comment (dfdx) [inline src/save.jl:220]: Should be in `load.jl`?

**Comment by dfdx** [comment]
Hi Duncan, great to see you here! This is a great addition to the codebase, thanks a lot for it. To make it available for a broader audience, let's first figure out a few things. 

1. I see that the package fails to instantiate on Julia 1.10+ due to outdated dependencies. It's on me to fix them (once I have time), but I wonder what Julia version you use? 
2. The goal of tests is to make a package robust to changes - without them we would have to manually test every function after every change, which, you can imagine, is quite cumbersome. For most functions, adding a test should be as easy as defining sample data and calling `ort_test()` ([example](https://github.com/FluxML/ONNX.jl/blob/master/test/saveload.jl#L17-L21)), but it's also possible to just include the sample graphs that you used for manual testing.
3. Since the diff is quite big, it will take me some time to analyze what exactly has been changed (e.g. reordering of functions vs. adding new definitions) and how it fits into broader ecosystem. If you find the the PR review takes longer than you'd expect, don't hesitate to make a few atomic PRs that I will be able to review and merge quickly.

File: src/save.jl (hunk 1 above)
Comment (dstarkenburg) [reply src/save.jl:220]: Whoops! My mistake.

**Comment by dstarkenburg** [comment]
Hey dfdx! (loving the username; it did take me a second to derive it) I realize I was in 1.10 when compiling but since have updated to 1.11. I will work on creating tests and perhaps create some atomic PRs with robust tests (i.e. one new supported layer at a time). Thank you for being receptive to additons; I look forward to working more on this package!

**Pull request closed without merging**